guides/source/action_cable_overview.md

```diff
@@ -263,14 +263,14 @@ WebSocket is opened.
 
 ```js
 // Specify a different URL to connect to
-createConsumer('https://ws.example.com/cable')
+createConsumer('ws://example.com/cable')
```

Comment:
I can see this can be confusing, but this example tries to show an example of overriding the configured URL and even using HTTP.
And WebSockets should work over HTTP:
> Although they are different, [RFC](https://en.wikipedia.org/wiki/RFC_(identifier)) [6455](https://datatracker.ietf.org/doc/html/rfc6455) states that WebSocket "is designed to work over HTTP ports 443 and 80 as well as to support HTTP proxies and intermediaries", thus making it compatible with HTTP.

https://en.wikipedia.org/wiki/WebSocket

What do you think about showing 2 separate examples?
```suggestion
createConsumer('wss://example.com/cable')
// Or when using websockets over HTTP
createConsumer('https://ws.example.com/cable')
```

Reply:
Thanks for the suggestion @p8 I will make the change to indicate the 2 separate examples.